buf/registry/module/v1beta1/branch_service.proto

```diff
@@ -98,3 +115,132 @@ message ListBranchesResponse {
   // The listed Branches.
   repeated Branch branches = 2;
 }
+
+message PushRequest {
+  message Value {
+    // The Branch to push the Commits to.
+    //
+    // If this is a name reference, and the named Branch does not exist, it will
+    // be created and the given Commits will be the first Commits on the Branch.
+    BranchRef branch_ref = 1 [(buf.validate.field).required = true];
+    // The ID of the Commits to push to the Branch, in the order they should be added.
+    //
+    // This Commit will become the head Commit on the Branch.
+    //
+    // If the Commit already exists on the Branch, an error is returned.
```

Comment:
Docs need updating now that this is a repeated field
```suggestion
    // If any of these Commits already exist on the Branch, an error is returned.
```

Reply:
Yes